### DISCUSSION TITLE
Dynamic skinning in lazer

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21424] [Q&A] [answered]
is lazer going to see possible dynamic skinning options (probably better way to word this)
for example: 
cursor trails that change colours
https://youtu.be/DAE9wsz3v8w

or other such features. currently these are only accessible in a std client called osufx, but having it come to lazer natively would be amazing.

**Comment by frenzibyte** [comment, marked as answer]
This will become possible once elements scripting is shipped in. See https://github.com/ppy/osu-framework/issues/3056.
